src/parseOption.ts

```diff
@@ -55,11 +55,21 @@ export function prepareCustomFieldsData(itemData: ScenegraphData, options: Optio
   const tooltipData: TooltipData[] = [];
 
   options.fields.forEach(function (fieldOption) {
+    const fieldStr = typeof fieldOption.field !== 'function' ? fieldOption.field : undefined;
```

Comment:
Let's use vega-util's `isFunction` so it's easier to read.  :)

(Same for other occurences of type check.)